microsoft-edge/extensions-chromium/developer-guide/declare-permissions.md

```diff
@@ -24,27 +24,91 @@ ms.date: 03/17/2021
 "permissions": [
   "tabs",
   "bookmarks",
+  "storage"
+],
+"optional_permissions": [
+  "unlimitedStorage"
+],
+"host_permissions": [
   "http://www.blogger.com/",
-  "http://*.google.com/",
+  "http://*.google.com/"
+]
+```
+
+#### [Manifest V2](#tab/v2)
+
+```json
+"permissions": [
+  "tabs",
+  "bookmarks",
+  "storage",
+  "http://www.blogger.com/",
+  "http://*.google.com/"
+],
+"optional_permissions": [
   "unlimitedStorage"
 ]
 ```
 
+---
+
+<!-- ====================================================================== -->
+
+Here's an example of the permissions part of a manifest file:
+
+#### [Manifest V3](#tab/v3)
+
+```json
+"permissions": [
+  "tabs",
+  "bookmarks",
+  "storage"
+],
+"optional_permissions": [
+  "unlimitedStorage"
+],
+"host_permissions": [
+  "http://www.blogger.com/",
+  "http://*.google.com/"
+],
+```
+
+#### [Manifest V2](#tab/v2)
+
+```json
+"permissions": [
+  "tabs",
+  "bookmarks",
+  "storage",
+  "http://www.blogger.com/",
+  "http://*.google.com/"
+],
+"optional_permissions": [
+  "unlimitedStorage"
+]
+```
+
+---
+
+
+<!-- ====================================================================== -->
 The following table lists the currently available permission strings to use in your manifest, and the descriptions.
 
 | Permission string | Details |
 |:--- |:--- |
 | `activeTab` | Requests that the extension is granted permissions according to the `activeTab` specification. |
 | `alarms` | Gives your extension access to the `chrome.alarms` API. |
-| `background` | Makes Microsoft Edge start up early and shut down late, so that extensions may have a longer life.  When any installed extension has `background` permission, Microsoft Edge runs invisibly as soon as the user logs into the user's computer, and before the user launches Microsoft Edge.  The `background` permission also makes Microsoft Edge continue running, even after its last window is closed, until the user explicitly quits Microsoft Edge. The `background` permission is normally used on a background page. <br/>Disabled extensions are treated as if they aren't installed. You should use the "background" permission with [background scripts](https://developer.chrome.com/docs/extensions/mv3/background_pages/).|
+| `background` | Makes Microsoft Edge start up early and shut down late, so that extensions may have a longer life. <br/>The `background` permission is normally used on a background page.  When any installed extension has `background` permission, Microsoft Edge runs invisibly as soon as the user logs into the user's computer, and before the user launches Microsoft Edge.  The `background` permission also makes Microsoft Edge continue running, even after its last window is closed, until the user explicitly quits Microsoft Edge. <br/>**Note:** Disabled extensions are treated as if they aren't installed. <br/>You should use the "background" permission with [background scripts](https://developer.chrome.com/docs/extensions/mv3/background_pages/).|
```

Comment:
<br/>The `background` permission is normally used on a background page.

This line is not needed because we are already adding it after the Note.

Reply:
Removed sentence.